lib/tasks/adhoc.rake

```diff
@@ -0,0 +1,15 @@
+namespace :adhoc do
+  desc 'Extract a list of All articles and what categories they sit in'
+  task :extract_page => :environment do
+    File.open('pages_with_cat.csv','w') do |file|
+      file.write "Url\tCategories\n"
+      en = Comfy::Cms::Site.where(label: :en).first
```

Comment:
I know they're the same and I'm not precious about it either, but if you have a tool made specifically for a particular job seems to me you might as well use it when the situation arises.